src/amo/api/collections.js

```diff
@@ -28,42 +32,153 @@ export const getCollectionDetail = (
 
 type GetCollectionAddonsParams = {|
   ...GetCollectionParams,
+  nextURL?: string,
   page?: number,
 |};
 
 export const getCollectionAddons = (
-  { api, page, slug, user }: GetCollectionAddonsParams
-) => {
+  { api, nextURL, page, slug, user }: GetCollectionAddonsParams
+): Promise<PaginatedApiResponse<ExternalCollectionAddon>> => {
   if (!slug) {
     throw new Error('slug is required');
   }
   if (!user) {
     throw new Error('user is required');
   }
 
-  return callApi({
+  const request = {
     auth: true,
-    endpoint: `accounts/account/${user}/collections/${slug}/addons`,
-    params: { page },
+    endpoint:
+      nextURL || `accounts/account/${user}/collections/${slug}/addons`,
+    params: undefined,
```

Comment:
Huh, I guess this is fine but it seems weird to pass `undefined`. Could we just leave this blank or use `null`? Just seems odd, is all.

Reply:
It's necessary so that Flow knows that `params` is a valid key. It actually makes a bit of sense too because adding `request.param` when it's uninitialized could be a mistake.

Reply:
Would it be okay to use `null` instead? It just seems weird to state a key as `undefined`.

Reply:
It has to be `undefined` so that the default value for `params` is used when `callApi` is invoked.

In fact, Flow will help you not make this mistake:

```
Error: src/amo/api/collections.js:60
 60:   return callApi(request);
                      ^^^^^^^ object literal. This type is incompatible with the expected param type of
 94: }: CallApiParams): Promise<any> {
        ^^^^^^^^^^^^^ object type. See: src/core/api/index.js:94
  Property `params` is incompatible:
     53:     params: null,
                     ^^^^ null. This type is incompatible with
     77:   params?: Object,
                    ^^^^^^ object type. See: src/core/api/index.js:77
```